pkg/koordlet/runtimehooks/hooks/groupidentity/rule.go

```diff
@@ -131,8 +132,12 @@ func (b *bvtPlugin) ruleUpdateCb(pods []*statesinformer.PodMeta) error {
 	for _, kubeQOS := range []corev1.PodQOSClass{
 		corev1.PodQOSGuaranteed, corev1.PodQOSBurstable, corev1.PodQOSBestEffort} {
 		bvtValue := r.getKubeQOSDirBvtValue(kubeQOS)
-		kubeQOSCgroupPath := koordletutil.GetKubeQosRelativePath(kubeQOS)
-		if err := sysutil.CgroupFileWrite(kubeQOSCgroupPath, sysutil.CPUBVTWarpNs, strconv.FormatInt(bvtValue, 10)); err != nil {
+		kubeQOSCgroupPath := koordletutil.GetPodQoSRelativePath(kubeQOS)
+		bvtUpdater, err := resourceexecutor.NewCommonCgroupUpdater(sysutil.CPUBVTWarpNs.ResourceType(), kubeQOSCgroupPath, strconv.FormatInt(bvtValue, 10))
+		if err != nil {
+			klog.Infof("bvtupdater creat failed, dir %v, error %v", kubeQOSCgroupPath, err)
+		}
+		if err := bvtUpdater.Update(); err != nil {
```

Comment:
If we expect to reduce the audit logs in the runtime hook plugins (otherwise, we can leave a TODO here and do it in another PR), we may need an Executor to do cacheable updates.